spring-cloud-config-server/src/test/java/org/springframework/cloud/config/server/environment/JGitConfigServerTestData.java

```diff
@@ -90,7 +101,7 @@ public static JGitConfigServerTestData prepareClonedGitRepository(Class... sourc
         //setup local repository
         File clonedRepoDir = new File("target/repos/cloned");
         if(clonedRepoDir.exists()) {
-            FileSystemUtils.deleteRecursively(clonedRepoDir);
+            FileUtils.deleteDirectory(clonedRepoDir);
```

Comment:
Please revert to `FileSystemUtils.deleteRecursively`

Reply:
ok, didn't work for my windows ...